pgqueuer/queries.py

```diff
@@ -543,6 +544,13 @@ async def clear_queue(self, entrypoint: str | list[str] | None = None) -> None:
             else self.driver.execute(self.qb.create_truncate_queue_query())
         )
 
+    async def mark_job_as_cancelled(self, ids: list[models.JobId]) -> None:
+        """
+        Clears jobs from the queue, optionally filtering by entrypoint if specified.
```

Comment:
The part of this doc comment referring to entrypoint seems to be incorrect?

Reply:
Yeah, i had an idea that it might make sense to clear by entrypoint as well?

Reply:
I could see that being useful, and it's consistent with the `clear_queue` method.